.github/actions/build-and-deploy/action.yml

```diff
@@ -0,0 +1,40 @@
+name: "Build and deploy neon-is-group-loader module"
+description: "Builds the service, pushes the image to Artifact Registry"
```

Comment:
use "Builds the module" instead of "service"

Reply:
Changed to 
"name: "Build a module and push the image"
description: "Builds the module, pushes the image to Artifact Registry"